utils/vara-ui/package.json

```
@@ -25,8 +25,8 @@
     "preview": "vite preview"
   },
   "peerDependencies": {
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react": "18.3.1",
+    "react-dom": "18.3.1"
```

```suggestion
    "react": "^18.2.0",
    "react-dom": ^18.2.0"
```

We can't be sure that every our app operates under `18.3.1`, therefore `peerDeps` should remain intact.